src/main/sensors/gyro.c

```diff
@@ -996,7 +996,7 @@ static FAST_CODE void checkForYawSpin(timeUs_t currentTimeUs)
     } else {
 #ifndef SIMULATOR_BUILD
         // check for spin on yaw axis only
-         if (abs(gyro.gyroADCf[Z]) > gyroConfig()->yaw_spin_threshold) {
+         if (fabsf(gyro.gyroADCf[Z]) > gyroConfig()->yaw_spin_threshold) {
```

Comment:
It's technically correct though as we're comparing to an integer and truncating the float gyro rate to an integer is fine in this case.